frontend/src/components/Header.js

```diff
@@ -34,6 +34,7 @@ export default function Header({ children, user }) {
 						<Typography variant="h6" component="div" sx={{ mr: 2, flexGrow: 1 }} onClick={() => routeChange("/faqs")}  style={{ cursor: "pointer" }}>
 							FAQS
 						</Typography>
+						<Button color="inherit" component={Link} to="/wallets">Wallets</Button>
```

Comment:
This button should only be visible if the user is logged in. Right now it throws an exception if the user is not logged in when the user clicks this button. I think later we will probably move this into the profile and thus it should not be a problem, but for now maybe do something similar as a few lines below, i.e. check if user is null and return <></> if this is the case or otherwise return the wallets button?